test/screenshot/lib/controller.js

```diff
@@ -84,6 +91,7 @@ class Controller {
 
   async initialize() {
     this.baseUploadDir_ = await this.storage_.generateUniqueUploadDir();
+    await this.gitRepo_.fetch();
```

Comment:
Also FYI this is going to conflict with the `shouldBuild` code that was merged.

Reply:
Fixed merge conflict. Thanks for the heads up!